android/app/src/main/java/ru/spbstu/amd/learnbraille/screens/lessons/InputDotsFragment.kt

```diff
@@ -44,10 +44,8 @@ class InputDotsFragment : AbstractLesson(R.string.lessons_help_input_dots) {
 
         val step = getStepArg()
         require(step.data is InputDots)
-        titleTextView.text = step.title
-        infoTextView.text = step.data.text
-            ?: getString(R.string.lessons_input_dots_info_template)
-                .format(step.data.dots.spelling)
+        //titleTextView.text = step.title
+        letter.text = " "
```

Comment:
We do not have a letter here. We have some message like "All dots" or generated one "Dots: 1 3 5"

Reply:
It should be printed in TitleTextView in smaller font